pkg/TorchVision/TorchVision.nupkgproj

```diff
@@ -17,5 +17,7 @@
     <Content Include="..\common\NormalPackage.targets" Pack="true" PackagePath="buildTransitive\netstandard2.0\$(MSBuildProjectName).targets" />
     <Content Include="..\empty.txt" Pack="true" PackagePath="lib\netstandard2.0\_._" />
     <Content Include="$(RepoRoot)\LICENSE.txt" Pack="true" PackagePath="" />
+    <PackageReference Include="SkiaSharp" Version="2.88.3" />
+    <PackageReference Include="SkiaSharp.NativeAssets.Linux" Version="2.88.3" />
```

Comment:
Is there a reason we choose Linux rather than [all](https://www.nuget.org/packages?q=SkiaSharp.NativeAssets) or nothing?

Reply:
cc @tarekgh

Reply:
The base SkiaSharp package contains MacOS and Windows assets already.

Reply:
@NiklasGustafsson is right. We do the same in ML. I would recommend using [SkiaSharp.NativeAssets.Linux.NoDependencies](https://www.nuget.org/packages/SkiaSharp.NativeAssets.Linux.NoDependencies) for Linux though. It is smaller and will not lose anything already using.

Reply:
Yes, thank you for pointing that out. That's what we should be using.